save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/InputForms.kt

```diff
@@ -87,8 +93,14 @@ internal fun RBuilder.inputTextFormRequired(
                 }
 
                 if (!validInput) {
-                    div("invalid-feedback d-block") {
-                        +"Please input a valid ${form.str}"
+                    if (isProjectOrOrganizationName) {
+                        div("invalid-feedback d-block") {
+                            +"Please input a valid ${form.str}. The name must not be longer than 64 characters and contain spaces."
```

Comment:
```suggestion
                            +"Please input a valid ${form.str}. The name can be no longer than 64 characters and can't contain any spaces."
```
I think it sounds a little bit better